src/main/webapp/state-info.js

```diff
@@ -18,9 +18,9 @@ const stateInfo = {
     '06': {
         'name': 'California',
         'ISO': 'US-CA',
-        lat: 38,
-        lng: -120,
-        zoomLevel: 6,
+        'lat': 38,
```

Comment:
Thanks for fixing these! I'll make sure to add them for all the states so that the geoJson can load for all states.